ads_plugin.py

```diff
@@ -432,11 +433,11 @@ def on_save_segmentation_button(self, event):
         myelin_array = myelin_array * params.intensity['binary']
         axon_array = axon_array * params.intensity['binary']
 
-        # Save the arrays as PNG files
+
         myelin_and_axon_array = (myelin_array // 2 + axon_array).astype(np.uint8)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg.png", img=myelin_and_axon_array)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg-myelin.png", img=myelin_array)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg-axon.png", img=axon_array)
+        ads_utils.imwrite(filename=save_dir + "/" + self.png_image_name[0][:-4] + axonmyelin_suffix.name, img=myelin_and_axon_array)
+        ads_utils.imwrite(filename=save_dir + "/" + self.png_image_name[0][:-4] + myelin_suffix.name, img=myelin_array)
+        ads_utils.imwrite(filename=save_dir +"/" + self.png_image_name[0][:-4] + axon_suffix.name, img=axon_array)
```

Comment:
`self.png_image_name` contains the name of every image that has been loaded in a session. You might get some unexpected file names after saving if you have loaded multiple images in FSLeyes. You could use the name of the overlay instead.

Reply:
@Stoyan-I-A thanks for your review; I will change it and ask for your review in PR#441. It would be great if you review in PR #441 since this PR is only meant for resolving the ADS dependency issue. 

Yeah, we should avoid slicing the image name and that was one of the reasons I mentioned switching to **Pathlib**. We are planning to first merge #441 and then #452, so that is why I used the used hardcoded slicing ([:-4)) which is currently being used in **ads_plugin.py**. I think it should be dealt with in #448 by using **.stem** for the path objects after merging PR's #441 and  #452.